lib/auth/authclient/authclient.go

```diff
@@ -55,6 +55,8 @@ func Connect(ctx context.Context, cfg *Config) (auth.ClientI, error) {
 		Credentials: []apiclient.Credentials{
 			apiclient.LoadTLS(cfg.TLS),
 		},
+		// Deliberately ignore HTTP proxies for backwards compatibility.
+		IgnoreHTTPProxy: true,
```

Comment:
I'm probably missing something, but I'm still not sure why we need to ignore it in the authclient too. Isn't lib/reversetunnel/resolver enough?

Reply:
It isn't enough. authclient was also affected by the original HTTP_PROXY change. `TestMultiPortNoProxy` doesn't pass unless authclient ignores proxies.